modules/cluster-logging-deploy-multitenant.adoc

```diff
@@ -5,26 +5,29 @@
 [id="cluster-logging-deploy-multitenant_{context}"]
 = Installing OpenShift Logging into a multitenant network
 
-If you are deploying OpenShift Logging into a cluster that uses multitenant isolation mode, projects are isolated from other projects. As a result, network traffic is not allowed between pods or services in different projects.
+If you deploy OpenShift Logging into a cluster that uses multitenant isolation mode, projects are isolated from other projects. As a result, network traffic is not allowed between pods or services in different projects.
 
 Because the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator are installed in different projects, you must explicitly allow access between the  `openshift-operators-redhat` and `openshift-logging` projects. How you allow this access depends on how you configured multitenant isolation mode.
 
 .Procedure
 
-To allow traffic between the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator, perform one of the following:
+To allow traffic between the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator, perform either of the following actions:
 
-* If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *Multitenant* mode, use the following command to join the two projects:
-+
-For example:
+* If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *Multitenant* mode, join the two projects. For example:
```

Comment:
@rolfedh, fyi, for consistency we've been calling these _network_ isolation modes: https://docs.openshift.com/container-platform/4.7/networking/openshift_sdn/about-openshift-sdn.html#nw-openshift-sdn-modes_about-openshift-sdn

Reply:
Thank you, @jboxman. This turned out to be quite important. The original content misused the terminology and was therefore incorrect.